nixos/modules/virtualisation/oci-containers.nix

```
@@ -280,6 +284,43 @@ let
           '';
         };
 
+        podman = mkOption {
+          type = types.nullOr (
+            types.submodule {
+              options = {
+                sdnotify = mkOption {
+                  default = "conmon";
+                  type = types.enum [
+                    "conmon"
+                    "healthy"
+                    "container"
+                  ];
+                  description = ''
+                    The way how `podman` should notify systemd that the unit is ready. There are
```

Small suggestion to improve clarity:

```suggestion
                    Determines how `podman` should notify systemd that the unit is ready. There are
```